.github/workflows/ci-failures-report.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Failures Report
+on:
+  workflow_dispatch:
+    inputs:
+      test:
+        description: Slack to #acs-slack-integration-testing
+        required: true
+        default: false
+        type: boolean
+  schedule:
+  - cron: 0 12 * * 1
+
+jobs:
+  report-e2e-failures-to-slack:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.61
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Report
+      env:
+        GCP_SERVICE_ACCOUNT_STACKROX_CI: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+        SLACK_CI_INTEGRATION_TESTING_WEBHOOK: ${{ secrets.SLACK_CI_INTEGRATION_TESTING_WEBHOOK }}
+        SLACK_ENG_DISCUSS_WEBHOOK: ${{ secrets.SLACK_ENG_DISCUSS_WEBHOOK }}
+      shell: bash
+      run: |
+        source scripts/ci/lib.sh
+
+        setup_gcp
+
+        # Avoid BQ init message in first report
+        bq --quiet --format=pretty query --use_legacy_sql=false 'select 1'
```

Comment:
will dry run do the job?

Reply:
I tried it. `--dry_run` does not create the `.bigqueryrc` file, so without this first run the first report will still have "Welcome to BigQuery" in the output. https://github.com/stackrox/stackrox/actions/runs/7226259185/job/19691439321?pr=9036#step:4:32